examples/tts/conf/hifigan/hifigan.yaml

```diff
@@ -0,0 +1,83 @@
+name: "HifiGan"
+train_dataset: ???
+validation_datasets: ???
+
+defaults:
+  - model/generator: v1
+
+model:
+  train_ds:
+    dataset:
+      _target_: "nemo.collections.tts.data.datalayers.AudioDataset"
+      manifest_filepath: ${train_dataset}
+      max_duration: null
+      min_duration: 0.75
+      n_segments: 8192
+      trim: false
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 16
+      num_workers: 4
+
+  validation_ds:
+    dataset:
+      _target_: "nemo.collections.tts.data.datalayers.AudioDataset"
+      manifest_filepath: ${validation_datasets}
+      max_duration: null
+      min_duration: 0.75
```

Comment:
Let's set min_duration to null for validation

Reply:
ok, i took this config from melgan [here](https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/melgan.yaml#L31). i will fix that

Reply:
Yea melGAN properly also needs this fix then.

Reply:
no problem